.travis.yml

```diff
@@ -2,7 +2,7 @@ dist: xenial
 sudo: true
 language: python
 python:
-  - '3.7'
+  - '3.6'
```

Comment:
Should we leave this?

Reply:
Yes, changed to 3.7.  I also changed another 3.6 to 3.7.